src/connector/src/sink/file_sink/s3.rs

```diff
@@ -82,26 +86,20 @@ impl<S: OpendalSinkBackend> FileSink<S> {
 
         if let Some(access) = config.common.access {
             builder = builder.access_key_id(&access);
-        } else {
-            tracing::error!(
-                "access key id of aws s3 is not set, bucket {}",
-                config.common.bucket_name
-            );
         }
 
         if let Some(secret) = config.common.secret {
             builder = builder.secret_access_key(&secret);
-        } else {
-            tracing::error!(
-                "secret access key of aws s3 is not set, bucket {}",
-                config.common.bucket_name
-            );
         }
 
         if let Some(assume_role) = config.common.assume_role {
             builder = builder.role_arn(&assume_role);
         }
-        builder = builder.disable_config_load();
+        // Default behavior is disable loading config from environment.
+        if !config.common.enable_config_load.unwrap_or(false) {
+            builder = builder.disable_config_load();
+        }
```

Comment:
Please extract a method and consider the env DISABLE_DEFAULT_CREDENTIAL like this https://github.com/risingwavelabs/risingwave/blob/436862f37b0963302af25d21913f7f06866fc6bc/src/connector/src/connector_common/iceberg/mod.rs#L165-L171

Reply:
The cloud side would set it.